tests/rotation.py

```diff
@@ -12,6 +12,55 @@
 from loguru import logger as LOG
 
 
+@reqs.description("Replace all nodes in a single transaction")
+@reqs.at_least_n_nodes(3)
+def test_replace_all_nodes(network, args):
+    current_nodes = network.get_joined_nodes()
+    current_node_ids = [node.node_id for node in current_nodes]
+
+    def make_node():
+        node = network.create_node("local://localhost")
+        network.join_node(node, args.package, args, timeout=3)
+        return node
+
+    new_nodes = [make_node(), make_node(), make_node()]
```

Comment:
Is there a reason we always create 3 nodes here, rather than `len(current_nodes)`?

Reply:
Not a good reason, but a path dependency because I think we started with two nodes and the next test needs f=1 or something like that, will fix.